python/test/unit/language/test_frontend.py

```diff
@@ -393,63 +394,6 @@ def test_constexpr_getitem():
     tl.arange(4, sum)
 
 
-@triton.constexpr_function
-def make_constexpr_closure(x):
-    x = tl.constexpr(x)
-
-    @triton.jit
-    def inner(shape: tl.constexpr):
-        return tl.full(shape, x, dtype=tl.int32)
-
-    return inner
-
-
-@filecheck_test
-@triton.jit
-def test_constexpr_closure():
-    # CHECK-LABEL: test_constexpr_closure
-    closure: tl.constexpr = make_constexpr_closure(42)
-
-    # CHECK: arith.constant dense<42> : tensor<128x128xi32>
-    closure((128, 128))
-
-
-@triton.constexpr_function
-def make_constexpr_generator(f):
-    f = tl.constexpr(f)
-
-    @triton.jit
-    def inner(lhs):
-        return lhs + f(lhs.shape, lhs.dtype)
-
-    return inner
-
-
-@triton.jit
-def inner_function(shape: tl.constexpr, dtype: tl.constexpr):
-    return tl.full(shape, 42, dtype)
-
-
-@filecheck_test
-@triton.jit
-def test_constexpr_generator():
-    # CHECK: func public @test_constexpr_generator
-    # CHECK:   [[RANGE:%.*]] = tt.make_range {end = 128 : i32, start = 0 : i32}
-    # CHECK:   call @{{.*}}make_constexpr_generator.<locals>.inner{{.*}}([[RANGE]])
-
-    # CHECK: func private @{{.*}}make_constexpr_generator.<locals>.inner
-    # CHECK:   [[RHS:%.*]] = tt.call @{{.*}}inner_function
-    # CHECK:   [[RESULT:%.*]] = arith.addi %arg0, [[RHS]]
-    # CHECK:   return [[RESULT]]
-
-    # CHECK: func private @{{.*}}inner_function
-    # CHECK:   %cst = arith.constant dense<42> : tensor<128xi32>
-    # CHECK:   return %cst
-    generator: tl.constexpr = make_constexpr_generator(inner_function)
```

Comment:
FYI these seemingly unrelated changes are because we're now computing the hash for all functions, which previously didn't happen if you were only running the parser. These examples didn't actually work if you tried them in a real kernel.